Jenkinsfile

```diff
@@ -1,63 +1,51 @@
-pipeline { 
-    agent none 
+pipeline {
+    agent none
     environment {
         ISISROOT="${workspace}" + "/build/"
         ISIS3TESTDATA="/usgs/cpkgs/isis3/testData/"
         ISIS3DATA="/usgs/cpkgs/isis3/data/"
     }
     stages {
-        stage('Fedora25') {
-            agent {
-                docker {
-                    label 'cmake'
-                    image 'chrisryancombs/docker_isis'
-                    args  '''\
-                            -v /usgs/cpkgs/isis3/data:/usgs/cpkgs/isis3/data \
-                            -v /usgs/cpkgs/isis3/testData:/usgs/cpkgs/isis3/testData\
-                            -v /usgs/cpkgs/isis3/isis3mgr_scripts:/usgs/cpkgs/isis3/isis3mgr_scripts
```

Comment:
I imagine these are being mounted by default now?

Reply:
Yeah inside the build containers. You can see them under Docker Templates in the Jenkins system config.